src/component/project/Project/Project.tsx

```diff
@@ -80,26 +86,84 @@ const Project = () => {
         /* eslint-disable-next-line */
     }, []);
 
+    const StyledDiv = styled('div')(({ theme }) => ({
+        display: 'flex',
+    }));
+
+    const StyledName = styled('div')(({ theme }) => ({
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
+        whiteSpace: 'nowrap',
+        paddingBottom: theme.spacing(2),
+    }));
+
+    const StyledTitle = styled('span')(({ theme }) => ({
+        fontSize: theme.fontSizes.smallBody,
+        fontWeight: 'normal',
+    }));
+    const StyledText = styled(StyledTitle)(({ theme }) => ({
+        color: theme.palette.grey[800],
+    }));
+
```

Comment:
Styled components should be moved to the top of the file, outside the component.

Reply:
Done